### ISSUE TITLE
Optional: Generate results in Latex

### ISSUE THREAD

**Issue by firefly-cpp** [lukapecnik/NiaAML#41]
Many researchers prefer the way that software supports generating results in Latex. So it is easy
to include results in research papers. If possible, please add this functionality.

**A word of warning:** I advise you not to use any of existing software packages for generating Latex from Python. 
From my past experience, many upstreams abandoned the development of their packages after a couple of releases.

**Comment by sisco0** [comment]
Have a good day @firefly-cpp. I was asking myself what the expected output would be like.
For this purpose I created an _Overleaf_ document which I am sharing here:
https://www.overleaf.com/read/dhvbqthmxpfy
Would that be the expected format like? If not, how should it be?
![image](https://user-images.githubusercontent.com/25695302/103063815-0bb90900-4578-11eb-96eb-33667b91919c.png)

**Comment by sisco0** [comment]
@LaurenzBeck , @firefly-cpp , does the current `NiaAML` package support any other kind of output format that is easily readable, such as _Markdown_?

**Comment by firefly-cpp** [comment]
@sisco0, not really.  What about generating a diagram using the Graphviz package?
